scripts/tsconfig.json

```diff
@@ -0,0 +1,21 @@
+{
+  "compilerOptions": {
+    "module": "CommonJS",
+    "noEmit": true,
+    "tsBuildInfoFile": "./tsconfig.tsbuildinfo",
+    "incremental": true,
+    "target": "ES5",
+    "jsx": "react",
+    "allowJs": true,
+    "pretty": true,
+    "strictNullChecks": true,
+    "noUnusedLocals": false,
+    "esModuleInterop": true,
+    "strict": true,
+    "allowSyntheticDefaultImports": true,
+    "moduleResolution": "node",
+    "baseUrl": ".",
+    "skipLibCheck": true,
+    "resolveJsonModule": true,
+  }
+}
```

Comment:
Why do we need a tsconfig file in the scripts folder?

Reply:
This lets us run `ts-node ./scripts/xyx` without passing any overrides (We previously had a `yarn ts-node` script that added additional options)

This will likely be removed eventually as scripts get removed

Reply:
It also makes it clear that this tsconfig applies to the scripts. We could technically use the root one, but that's ambiguous